lib/google_contacts_api/groups.rb

```diff
@@ -23,5 +23,28 @@ def get_groups(params = {})
       end
       GoogleContactsApi::GroupSet.new(response.body, @api)
     end
+
+    def create_group(title)
```

Comment:
Nothing to fix here, but this made me realize I probably shouldn't have implemented this as a module, but maybe as a "fetcher" object outside of the user/group/contact hierarchy.